demo/src/main/java/hu/flowacademy/stockmarket/config/BeanDefinition.java

```diff
@@ -0,0 +1,33 @@
+package hu.flowacademy.stockmarket.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class BeanDefinition {
+
+    @Bean
+    public ObjectMapper getObjectMapper() {
+        return new ObjectMapper();
+    };
+
+    @Bean
+    public RestTemplate getRestTemplate() {
+        return new RestTemplate();
+    }
+
+    @Bean
+    public HttpHeaders getHttpHeaders() {
+        return new HttpHeaders();
+    }
+
+    @Bean
+    public Gson getGson() {
```

Comment:
Ha ObjectMapper-t használsz, akkor nem kell a Gson.

Reply:
Valamiért nem működött az ObjectMapper-rel való tömb beolvasás.

Reply:
Próbáld meg újra, mert mennie kell ObjectMapperrel és akkor meg tudunk spórolni egy függőséget.

Reply:
List<StockMonthly> list = Arrays.asList(objectMapper.readValue(obj.toString(), StockMonthly[].class));

A fenti kódrészletre dobja az alábbi exceptiont:
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "uHigh" (class hu.flowacademy.stockmarket.persistance.model.StockMonthly), not marked as ignorable (16 known properties: "high", "uclose", "open", "date", "uvolume", "change", "changeOverTime", "low", "changePercent", "uhigh", "close", "volume", "id", "uopen", "label", "ulow"])